### PR TITLE
adjust sku validation

### PR DESCRIPTION
**Pull request by rayssamfreitas**
## What's the purpose of this pull request?

Need to adjust the SKU validation because when selecting an SKU with the name "7", for example, the SKU with the name "7.5" is being selected instead of "7".

## How it works?

The adjustment involves replacing the code that previously used slugs.includes() with strict equality slugs ===

## How to test it?

On a product page with SKUs, if you select one named '7', you'll notice that it selects SKU '7.5' instead.

### CODE REVIEW

File: packages/components/src/molecules/SkuSelector/useSkuSlug.ts [hunk 1]
```diff
@@ -14,11 +14,11 @@ function getSkuSlug(
 
   const possibleVariants = Object.keys(slugsMap)
 
-  const firstVariationForDominantValue = possibleVariants.find((slug) =>
-    slug.includes(
-      `${dominantVariation}-${selectedVariations[dominantVariation]}`
+  const firstVariationForDominantValue = possibleVariants.find((slug) => {
+    return slug.includes(
+      `${dominantVariation}-${selectedVariations[dominantVariation]}-`
```
Comment (eduardoformiga) [inline packages/components/src/molecules/SkuSelector/useSkuSlug.ts:19]: these changes don't fix the problem.

**Review by eduardoformiga — COMMENTED** [review]
Thank you for opening this PR :)

In my tests, the changes from this PR don't fix the root problem. 

I'll open a new PR with some fixes in this logic and get back to you so that you can test the version, ok?

**Comment by eduardoformiga** [comment]
@rayssamfreitas, can you double-check if this version fixes the problem?

"@faststore/core": "https://pkg.csb.dev/vtex/faststore/commit/9ec25832/@faststore/core"

**Comment by rayssamfreitas** [comment]
@eduardoformiga, I've tested your version and it works! It resolves the issue. Thank you! I'll close this PR.

**Comment by eduardoformiga** [comment]
Nice! The faststore core version `3.0.77` was published with the fix 🎉

**Pull request closed without merging**